### ISSUE TITLE
Search results should include group names

### ISSUE THREAD

**Issue by RTechSn** [Kunzisoft/KeePassDX#1734]
**Is your feature request related to a problem? Please describe.**
Hi.
I'm organising my database by groups: for each machine there is a group with a distinct name. In that group there are passwords, that can have repeated names in other groups.
For example: 
MyComputer: 
- Admin
- User
- Something Else
MyMomsComputer
- Admin
- User

... etc

As you can see, if i search for something like "Admin" i'll get many results, which are hard to parse, since there is many simillar entries.
But if i search for "MyComputer", i get nothing.

**Describe the solution you'd like**
Search results should include group names. By clicking a group name result, respective group should be navigated to.

**Comment by J-Jamet** [comment]
You can search by group by going to the corresponding group and checking the current group.
I'm aware that this isn't the most intuitive way of doing things, but selecting groups to search for would require a tree selection with new screens. It's enviable, but not a priority, as the work involved is huge.
